db/wpdb.ts

```diff
@@ -626,6 +626,6 @@ export const flushCache = () => {
     cachedAuthorship = undefined
     cachedEntries = []
     cachedFeaturedImages = undefined
-    cachedPosts = undefined
+    getBlogIndex.cache.clear?.()
```

Comment:
Interesting!

I didn't know you could do that.

I had to test it myself:

`"foo".toUpperCase?.()`

Checks out :)